canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -22,6 +22,9 @@ $toolbar-button-height: 40px;
 $toolbar-button-width: 40px;
 $toolbar-divider-width: 1px;
 
+.properties-custom-table-buttons .toolbar-right-bar {
```

Comment:
`properties-custom-table-buttons` is a class name inside common properties. It should not be part of the toolbar styles. 

> Revisiting the table-buttons.scss, we actually still need to hide the right side toolbar from inside common properties tables

This change is specific to table-buttons and should not be a part of the toolbar. Having this in the toolbar styles is saying that any small sized toolbars will automatically hide the right side items. Can you move this to the table-buttons? Also lets keep the css selector to be more define 

```
.properties-custom-table-buttons {
	.toolbar-div {
		.toolbar-right-bar {
			display: none;
		}
	}
}
```

Reply:
same comment below